pom.xml

```diff
@@ -1575,6 +1589,16 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>jdk11plus</id>
+            <activation>
+                <jdk>[11,)</jdk>
+            </activation>
+            <properties>
+                <failsafe.argLine>--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.net=ALL-UNNAMED</failsafe.argLine>
+                <surefire.argLine>-Xshare:off ${failsafe.argLine}</surefire.argLine>
```

Comment:
-1, `-Xshare:off` will not work on all JDKs (like non-hotspot-JDKs as OpenJ9 which comes with IBM Semeru). Either additionally check for a hotspot JDK or just leave it out. It should not be required anyway.

Reply:
OpenJ9 (Semuru) works perfectly with `-Xshare:off`